aspnetcore/blazor/forms-validation.md

```diff
@@ -184,6 +243,93 @@ Create a component with the following markup, and use the component just as `Inp
         this, __value => CurrentValueAsString = __value, CurrentValueAsString)" />
 ```
 
+## Work with radio buttons
+
+When working with radio buttons in a form, data binding is handled differently than other elements because radio buttons are evaluated as a group. The value of each radio button is fixed, but the value of the radio button group is the value of the selected radio button. The following example shows how to handle data binding for a radio button group:
```

Comment:
```suggestion
When working with radio buttons in a form, data binding is handled differently than other elements because radio buttons are evaluated as a group. The value of each radio button is fixed, but the value of the radio button group is the value of the selected radio button. The following example shows how to handle data binding for a radio button group and add support for validation using a custom `InputRadio` component:
```